### PR TITLE
doc: reading note in December,2018

### PR DESCRIPTION
**Pull request by aermin**

### CODE REVIEW

File: _posts/2018-12-24-What-is-a-PWA-and-why-should-you-care.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  5.Splash Screen
+
+  PWA adds a splash screen during the startup of the app. This makes the PWA feel more like a native app.
+
+  6.Highly engage-able
+  
+  The app should keep the users engaged. A PWA provides features like push notification, home screen icon, full-screen and offline first app to glorify user engagement.
+
+  And not mater framework you use, PWA only needs the required components.
```
Comment (u9520107) [inline _posts/2018-12-24-What-is-a-PWA-and-why-should-you-care.md]: not mater => no matter what framework you use,

File: _posts/2018-12-24-What-is-a-PWA-and-why-should-you-care.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+  There are three components as follow which are required to develop a PWA.
```
Comment (u9520107) [inline _posts/2018-12-24-What-is-a-PWA-and-why-should-you-care.md]: There are three components required to develop a PWA as shown below:

File: _posts/2018-12-24-What-is-a-PWA-and-why-should-you-care.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  
+  The first component is the Service worker. Service worker works as a proxy between the browser and the network. It manages notifications pushing and helps to use web application in offline with the browser's cache API. 
+  
+  The second component is The manifest file. It is a config JSON file which contains the information of your application, such as the icon which is displayed on the home screen when installed, the short name of the application, background color, and theme.
+
+  The third component is HTTPS. PWA need secure protocol HTTPS because that the service worker has the ability to intercept the network requests and modify the responses
+
+  ### Reflection:
+  
+  Progressive web application (PWA) is created to make web app provide a native-like experience with some components for users. Such as enabling to load when there is no network or network id poor, installing this app and show in the home screen, opening this app with full screen, having notification pushing and so on.
```
Comment (u9520107) [inline _posts/2018-12-24-What-is-a-PWA-and-why-should-you-care.md]: network id poor => network is poor
installing this app => installing as app
opening this app with full screen => opening app in full screen

File: _posts/2018-12-24-What-is-a-PWA-and-why-should-you-care.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+  As far as I am concerned, I think PWA can play a more and more important part in the unified application development. There are some reasons as follows. On the one hand, web is open and the mobile native application is closed. It causes users to search app with low efficiency in fixed app stores. On the other hand, Progressive web applications developed by a set of web technologies can work on different platforms. It can reduce lots of development cost. Of cause, the premise is that Progressive web applications should provide as good experience as native applications providing. And It almost did now in most scenarios.
```
Comment (u9520107) [inline _posts/2018-12-24-What-is-a-PWA-and-why-should-you-care.md]: Just drop the 'As far as I am concerned'.
"there are some reasons as follows", I think you should drop this line too.
And It almost did now => And it almost does now

File: _posts/2018-12-24-What-is-a-PWA-and-why-should-you-care.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+  PWA deserves us to care because of the benefits.
```
Comment (u9520107) [inline _posts/2018-12-24-What-is-a-PWA-and-why-should-you-care.md]: deserves us to care => deserves our attention

Comment (aermin) [reply _posts/2018-12-24-What-is-a-PWA-and-why-should-you-care.md]: Thank you！ 😄

**Pull request merged**